modules/update-using-custom-machine-config-pools-inheritance.adoc

```diff
@@ -0,0 +1,206 @@
+// Module included in the following assemblies:
+//
+// * updating/updating_a_cluster/update-using-custom-machine-config-pools.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="update-using-custom-machine-config-pools-mcp-inheritance_{context}"]
+= Managing machine configuration inheritance for a worker pool canary
+
+You can configure a machine config pool (MCP) canary to inherit any `MachineConfig` assigned to an existing MCP. 
+This is useful if, for example, you want to use a canary MCP to test as you update the nodes of an existing MCP, one at a time.
+
+
+.Prerequisites
+
+* You have created one or more custom machine config pools (MCP).
+
+.Procedure
+
+. Create a secondary MCP. 
+.. Save the following file as `machineConfigPool.yaml`.
++
+[source,yaml]
+----
+apiVersion: machineconfiguration.openshift.io/v1
+kind: MachineConfigPool
+metadata:
+  name: worker-perf
+spec:
+  machineConfigSelector:
+    matchExpressions:
+      - {
+         key: machineconfiguration.openshift.io/role,
+         operator: In,
+         values: [worker,worker-perf]
+        }
+  nodeSelector:
+    matchLabels:
+      node-role.kubernetes.io/worker-perf: "" 
+----
++
+.. Create the new machine config pool by running the following command:
++
+[source,terminal]
+----
+$ oc create -f machineConfigPool.yaml
+----
++
+.Example output
++
+[source,terminal]
+----
+machineconfigpool.machineconfiguration.openshift.io/worker-perf created
+----
+
+. Add some machines to the secondary MCP.
+This example labels the worker nodes, `worker-b` and `worker-c` to the MCP `worker-perf`:
++
+[source,terminal]
+----
+$ oc label node worker-b node-role.kubernetes.io/worker-perf=''
+$ oc label node worker-c node-role.kubernetes.io/worker-perf=''
+----
+
+. Create a new `MachineConfig` for the MCP `worker-perf`.
+.. Save the following `MachineConfig` example as `new-machineconfig.yaml`.
++
+[source,yaml]
+----
+apiVersion: machineconfiguration.openshift.io/v1
+kind: MachineConfig
+metadata:
+  labels:
+    machineconfiguration.openshift.io/role: workers-perf
+  name: 06-kdump-enable-workers-perf
+spec:
+  config:
+    ignition:
+      version: 3.2.0
+    systemd:
+      units:
+      - enabled: true
+        name: kdump.service
+  kernelArguments:
+    - crashkernel=512M
```

Comment:
can this be configured? or is 512M the only valid value?

Reply:
I think this is just an example value. The important point comes up during verification where the user can check that this value has been set as specified.

Reply:
+1